src/test/java/ua/ithillel/lesson14/ThreadSafeListTest.java

```diff
@@ -0,0 +1,57 @@
+package ua.ithillel.lesson14;
+
+import org.junit.jupiter.api.Test;
+import ua.ithillel.lesson14.item3.ThreadSafeList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ThreadSafeListTest {
+    List<String> list = new ArrayList<>();
+    Thread threads = null;
+
+    @Test
+    public  void shouldAdd() throws InterruptedException {
+
+        for (int i = 0; i < 100; i++) {
+            String item = "item" + i;
+
+            threads = new Thread(new ThreadSafeList<>(list));
```

Comment:
```suggestion
            threads = new Thread(() -> list.add(item));
```
десь так можна спробувати це протестувати.